src/main/java/com/google/sps/servlets/quizTiming.java

```diff
@@ -0,0 +1,32 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.gson.Gson;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import java.util.List; 
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
```

Comment:
The following imports could likely be safely removed:

import com.google.gson.Gson;
import java.util.ArrayList;
import java.util.List; 
import com.google.appengine.api.users.UserService;
import com.google.appengine.api.users.UserServiceFactory;

Be sure to rebuild after removing these though to ensure they actually are not needed.